src/Authentication.Abstractions/Interfaces/IAuthenticationStore.cs

```diff
@@ -24,6 +24,12 @@ public interface IAzureTokenCache
         /// </summary>
         byte[] CacheData { get; set; }
 
+        /// <summary>
```

Comment:
We can'd add this interface - this would be a breaking change.  The best we can do is add an additional interface with this method and have AuthenticationStore implement both

Reply:
@markcowl added a new interface `IAzureMsalTokenCache`, moved this function to the interface and made `AuthenticationStore` implement it in addition to `IAzureTokenCache`